database/dataGen.js

```diff
@@ -35,7 +35,8 @@ const dataGen = (writer, encoding, callback) => {
         five: fives,
       };
       const numberOfRatings = ones + twos + threes + fours + fives;
-      const starPercentages = JSON.stringify(stars);
+      const starRatings = JSON.stringify(stars);
+      const starPercentages = starRatings.replace(/"/g, "'");
```

Comment:
Is this just a quirk of how postgres ingests the data that it dislikes double quotes?

Reply:
Yeah i had been hearing from other guys that it causes problems when parsing unless done this way